src/firestore/src/generated/gapic/builder.rs

```diff
@@ -17,7 +17,7 @@
 pub mod firestore {
     use crate::Result;
 
-    /// A builder for [Firestore][super::super::client::Firestore].
+    /// A builder for [Firestore][crate::client::Firestore].
```

Comment:
This is the case I was worried about. It would work because of:

https://github.com/googleapis/google-cloud-rust/blob/847a924abf9ac12c49b312e599a0a214f9215295/src/firestore/src/lib.rs#L35

I assume if we decided to split or rename the firestore client we can apply your new rename feature at the same time. SGTM.